Tests/test_file_libtiff.py

```diff
@@ -9,7 +9,7 @@
 import pytest
 
 from PIL import Image, ImageFilter, TiffImagePlugin, TiffTags, features
-from PIL.TiffImagePlugin import SUBIFD
+from PIL.TiffImagePlugin import SUBIFD, STRIPOFFSETS
```

Comment:
```suggestion
from PIL.TiffImagePlugin import STRIPOFFSETS, SUBIFD
```
Lint fix.

Reply:
Saw that, but didn't bother since the test is not working...